canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -306,6 +308,12 @@ $palette-border-color: $ui-03;
 	position: absolute;
 	overflow-x: hidden;
 	overflow-y: overlay;
+	li .bx--accordion__arrow {
+		margin-top: 0.2rem;
+	}
+	button::before .common-canvas.classic-vertical .palette-flyout-categories .tooltip-container{
```

Comment:
This selector is not valid. The use of .common-canvas.classic-vertical  will only work in the test harness. In general we should not have to use such long selectors to do the styling. What is this even doing?